docs/administration/configuration/hashicorp-consul.md

```diff
@@ -0,0 +1,53 @@
+# HashiCorp Consul (Enterprise)
+
+Rundeck Enterprise users have the option to use HashiCorp Consul as a method to edit Rundeck's configuration properties. By doing so, it allows users to manage the configuration for all of their services in one place. In terms of Rundeck, this feature is very beneficial because it allows users to edit configurations across all of their Rundeck instances. So, instead of having to change a configuration for each instance, you can change it once in Hashicorp and it will apply to all instances.  
+
+## Getting Started 
+
+To begin, we need to make sure that we have both Rundeck and HashiCorp Consul installed. The easiest way to get HashiCorp up and running is using their docker image. The docker image is named "consul." Consul can be found at localhost:8500, but this can also be changed from within Consul. 
+We also need to make sure that Rundeck knows to use Consul for configuration. There are a couple of different ways to do that:
+
+### RPM, Debian and Docker Users
+
+For RPM, Debian and Docker users, the best way to specify that Rundeck should use Consul, is by defining the environment variable to be true. To do so, we need to add the following to the `/etc/rundeck/profile` file:
+```
+RUNDECK_CONSUL_ENABLED=true
+```
+
+### War Users
+
+For customers who use installed Rundeck using the .war, there are a couple different options.
+1. You could define the environment variable just like RPM, Debian and Docker users by adding the following to the `/etc/rundeck/profile` file:
+2. You could pass the property as a flag when starting Rundeck. An example of that would look like the following:
+```
+java -Xmx4g -Drundeck.consul.enabled=true -jar rundeck-{{{rundeckVersionFull}}}.war
+```
+
+### Setting the Consul server in Rundeck
+
+Now that you have specified to use Consul, we can shut down Rundeck and look at the newly created `bootstap.yml` file, which is located in the ${rdbase}/server/config directory. In there, it will specify the host of Consul. The host property should be set to the ip address or hostname of your Consul server.
+
+## Services in Consul
+
+In HashiCorp, you will notice that Rundeck may not appear right away as a service. However, once the Rundeck service is fully up and running, it will show up in HashiCorp.
+
+## Key/Value in Consul
+
+This is where you can specify the various configurations for Rundeck. To begin, you need to create a folder called "config", with another folder called "rundeck" inside of it. Inside of the Rundeck folder is where all the Rundeck configuration will be placed. Begin by creating a key value named "data." Now, copy and paste all the contents in the `rundeck-config.properties` file into the value for that key. After that, comment out the entire `rundeck-config.properties` file. Now, when we relaunch Rundeck, it will pull the configuration settings from Consul. 
```

Comment:
Is it common to do this all in one consul setting versus individual settings?

Reply:
Yes, this was the change that Stephen made. Now, by default you will copy and paste the configuration instead of doing it one by one as that was a tedious process.